keycloak/themes/hmda/login/register.ftl

```diff
@@ -155,43 +110,22 @@
                 },
                 formatResult: function(institution) {
 
-                    return '<div class="container-fluid">' +
-                           '  <div class="row">' +
-                           '    <div styles="vertical-align:middle;" class="col-md-1">' +
-                           '      <h1>' + getStatusIcon(getFormEmail(), institution.domain[0]) + '</h1>' +
-                           '    </div>' +
-                           '    <div class="col-md-11">' +
-                           '      <div class="row">' +
-
-                           '        <div class="col-md-6">' +
-                           '          <div class="row">' +
-                           '            <div class="col-md-12">' +
-                           '              <h4>' + institution.name + '</h4>' +
-                           '            </div>' +
-                           '          </div>' +
-
-                           '          <div class="row">' +
-                           '            <div class="col-md-4">' +
-                           '               <i class="fa fa-gavel" aria-hidden="true"></i> ' + institution.regulator +
-                           '            </div>' +
-                           '            <div class="col-md-8">' +
-                           '               <i class="fa fa-envelope" aria-hidden="true"></i> ' + institution.domain[0] +
-                           '            </div>' +
-                           '          </div>' +
-
+                    return '<div class="usa-grid-full">' +
+                           //'    <div styles="vertical-align:middle;" class="col-md-1">' +
+                           //'      <h1>' + getStatusIcon(getFormEmail(), institution.domain[0]) + '</h1>' +
```

Comment:
Is there a plan to re-incorporate some form of status indicator?  Without this, there's no way for the user to know ahead of time if the institution they've selected is going to work.

I realize there's a bit less real estate now, so this is likely a question of _where do we put it?_